### DISCUSSION TITLE
List sorting

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#632] [Feature requests]
Add the ability to sort lists of things and save that to the DB similar to how you sort menu items in other admins.

![image](https://user-images.githubusercontent.com/1334409/152534882-edbf2393-cf5f-49cb-b3c3-c41dc129c408.png)

**Comment by JensRavens** [comment]
This would be super helpful with one of our current projects. E.g. taking the list of associations (here: comments of posts in the avo demo):
<img width="1608" alt="Screenshot 2022-03-02 at 18 57 48" src="https://user-images.githubusercontent.com/928979/156420591-cafa46c5-e9ec-420d-95fd-1d652cc2269e.png">

We're using [acts_as_list](https://github.com/brendon/acts_as_list) here to make them sortable (and adding custom actions on them to do that, but that's quite inconvenient). Would be great to support that gem out of the box or just support a `position` column. 

Drag&Drop of course is nice, but even just having position buttons ("move up", "move down") on each list entry would already greatly simplify the process for us.

**Reply by adrianthedev** [reply]
Thanks for the feedback. This is quite a requested feature time and time again. Let me see what I can do to add at least the basic up/down arrows to the rows

**Comment by adrianthedev** [comment]
Incoming soon
https://twitter.com/avo_hq/status/1499294651886489604

**Comment by adrianthedev** [comment]
Added in https://github.com/avo-hq/avo/pull/712

**Reply by adrianthedev** [reply]
https://www.youtube.com/watch?v=LEALfPiyfRk
